### ISSUE TITLE
Implement sounds for notifications and events

### ISSUE THREAD

**Issue by 20seven** [FAForever/downlords-faf-client#746]
Sounds for certain notifications/events aren't implemented yet. 

In AudioServiceImpl.java:

```
    // TODO implement
//    friendOnlineSound = loadSound(FRIEND_ONLINE_SOUND);
//    friendOfflineSound = loadSound(FRIEND_OFFLINE_SOUND);
//    friendJoinsGameSound = loadSound(FRIEND_JOINS_GAME_SOUND);
//    friendPlaysGameSound = loadSound(FRIEND_PLAYS_GAME_SOUND);
```

**Comment by Sheikah45** [comment]
@axel1200 is this still relevant?

**Comment by 1-alex98** [comment]
I think it is

**Comment by andrewtvuong** [comment]
Is this still relevant? Have no friends on FAF to test lol.. would be interested in feature request for sound to alert host or something, with cooldowns to avoid spamming. This would be helpful to wake up afk host waiting on 1/12 Dual Gap

**Comment by Sheikah45** [comment]
Yes, still relevant. The sounds have still not been implemented

**Comment by magge-faf** [comment]
I have no authorization to push features, so I just created an own GitHub project to showcase my implementation by following the GitHub branching standards & conventions:

https://github.com/magge-faf/downlords-faf-client/commit/841b57174ba8a1b7d5e05d31851261ea3e489b31

---

This was hard-coded:

`private static long SILENCE_PERIOD_AFTER_SOUND = 30000;
`

Best case scenario would be to let the user decide which threshold he wants:

I would add another fancy sub-menu under the existing "Enable sounds" where the user can have an input field for seconds to set his preferred silence:

![image](https://user-images.githubusercontent.com/101107758/212303742-ea8fb207-7a7d-4c53-8d5f-7405e161268f.png)

Is this a solution or had the developers a different approach in mind?

**Comment by Sheikah45** [comment]
I think that is a solution. Although I might try to find a better name than silence after sound.

Also you can make a PR with the sounds from your fork to develop

**Comment by magge-faf** [comment]
If you give me permission to PR, I would request those changes: https://github.com/magge-faf/downlords-faf-client/commit/841b57174ba8a1b7d5e05d31851261ea3e489b31

I would create later another issue/feature request about the users-input-threshold-menu. The foundation for the new notification sounds must have green light in the first place.

**Comment by Sheikah45** [comment]
Should be good to make the pr now

**Comment by Sheikah45** [comment]
Fixed in #2889
